src/components/partner/partner.service.ts

```diff
@@ -186,6 +193,28 @@ export class PartnerService {
     return await mapListResults(results, (dto) => this.secure(dto, session));
   }
 
+  async listProjects(
+    partner: Partner,
+    input: ProjectListInput,
+    session: Session
+  ): Promise<SecuredProjectList> {
+    const projectListOutput = await this.projectService.list(
+      { ...input, filter: { ...input.filter, partnerId: partner.id } },
+      session
+    );
+
+    const perms = await this.authorizationService.getPermissions({
+      resource: Partner,
+      sessionOrUserId: session,
```

Comment:
```suggestion
      sessionOrUserId: session,
      dto: partner,
```
Need to pass in partner object for it's sensitivity and scoped roles

Reply:
@CarsonF - Makes sense, but I added that and it is still coming back false.